### ISSUE TITLE
How to save file to disk?

### ISSUE THREAD

**Issue by jesusgn90** [Naltox/telegram-node-bot#169]
Here is my code:

```
        let document = $.message.document;
        (document) ? $.sendMessage('Greate i\'m going to convert it...') : $.sendMessage('Please send me a document...');

        if(document) {
            $.api.getFile(document.fileId).then(function(data){
                          // HOW I SAVE THIS FILE TO HARD DISK OR SOMEWHERE ELSE
            });
        }
```

**Comment by swantzter** [comment]
you should probably use node's inbuilt [fs](https://nodejs.org/api/fs.html)

**Comment by jesusgn90** [comment]
I know fs of course but how to pipe File type object in my code to fs?

**Comment by swantzter** [comment]
hmm, yeah I investigated a bit, and I initially thought `data` would contain the file's data in some way... it apparently didn't. however `data.file_path` can be used to download the file from https://api.telegram.org/file/bot<token>/<file_path> the file must be no larger than 20mb. see <https://core.telegram.org/bots/api#getfile>

**Comment by jesusgn90** [comment]
Yes I know it too but I think the path it's wrong the URL sends error if you try, let's try by yourself please

**Comment by jesusgn90** [comment]
Help me with this please I like this module I don't want to change it!
